recipe/meta.yaml

```diff
@@ -23,33 +23,32 @@ requirements:
     - python >=3.8,<3.11
     - pip
     - coiled
-    - numpy ==1.21.6
-    - pandas ==1.4.2
+    - numpy ==1.22.4
+    - pandas ==1.4.3
     - dask =={{ dask_version }}
     - distributed =={{ distributed_version }}
-    - fsspec ==2022.5.0
-    - s3fs ==2022.5.0
-    - gcsfs ==2022.5.0
+    - fsspec ==2022.7.1
+    - s3fs ==2022.7.1
+    - gcsfs ==2022.7.1
     - pyarrow ==8.0.0
-    - jupyterlab ==3.4.2
-    - dask-labextension ==5.2.0
+    - jupyterlab ==3.4.5
+    - dask-labextension ==5.3.0
     - lz4 ==4.0.0
-    - ipywidgets ==7.7.0
-    - numba ==0.55.1
-    - scikit-learn ==1.1.1
+    - ipywidgets ==7.7.1
+    - numba ==0.55.2
+    - scikit-learn ==1.1.2
     - ipycytoscape ==1.3.3
     - click ==8.1.3
-    - xarray ==2022.3.0
-    - zarr ==2.11.3
-    - msgpack-python ==1.0.3
+    - xarray ==2022.6.0
+    - zarr ==2.12.0
+    - msgpack-python ==1.0.4
     - cloudpickle ==2.1.0
     - tornado ==6.1
     - toolz ==0.11.2
     - python-blosc ==1.10.2
     - zict ==2.2.0
     - xgboost ==1.6.1
     - dask-ml ==2022.5.27
-    - openssl >1.1.0g
```

Comment:
Note that we're removing this `openssl` pin here as it's no longer needed with newer versions of `distributed` that contain https://github.com/dask/distributed/pull/6562